### ISSUE TITLE
Spec parser failing due to undefined nature in extension property

### ISSUE THREAD

**Issue by goneall** [spdx/spdx-3-model#580]
It looks like the spec parser has been failing for a while, but not reported in the CI due to https://github.com/spdx/spec-parser/issues/80

Here's the output of the run:
```
Run cd spec-parser
WARNING : External property restrictions aren't yet handled properly, they are added to the description of the class.
WARNING : External property restrictions aren't yet handled properly, they are added to the description of the class.
WARNING : External property restrictions aren't yet handled properly, they are added to the description of the class.
WARNING : skipping ../model/Core/Properties/_property.md
WARNING : skipping ../model/Core/Vocabularies/_vocab.md
WARNING : External property restrictions aren't yet handled properly, they are added to the description of the class.
WARNING : External property restrictions aren't yet handled properly, they are added to the description of the class.
WARNING : Overwriting out_dir `/tmp/spec-parser.out`
ERROR   : Invalid Nature attribute in metadata `None`
```

I've isolated this to the `extension` property in core which is missing a `Nature` field.

There is an outstanding pull request for `extension` which could fix this once other identified issues are resolved.

**Comment by goneall** [comment]
We might want to "Hack" the extensions property to add some kind of nature and range (e.g. just a string for now) so that the spec-parser will pass.

**Comment by goneall** [comment]
Fixed with PR #503
